buildSrc/src/main/java/Dependencies.kt

```diff
@@ -23,7 +23,7 @@ object Versions {
     const val lifecycle = "1.1.1"
 
     const val dokka = "0.9.17"
-    const val android_gradle_plugin = "3.2.1"
+    const val android_gradle_plugin = "3.3.2"
```

Comment:
Why did you need to update the gradle plugin version?

Reply:
I thought application-services had upgraded the gradle plugin, but it was just my local build.